internal/pkg/template/templates/cicd/buildspec.yml

```diff
@@ -51,9 +51,9 @@ phases:
       # The tag is the build ID but we replaced the colon ':' with a dash '-'.
       # We truncate the tag (from the front) to 128 characters, the limit for Docker tags
       # (https://docs.docker.com/engine/reference/commandline/tag/)
-      - tag=$(sed 's/:/-/g' <<<"$CODEBUILD_BUILD_ID" | rev | cut -c 1-128 | rev)
       - >
         for env in $pl_envs; do
+          tag=$(sed 's/:/-/g' <<<"${CODEBUILD_BUILD_ID}-${env}" | rev | cut -c 1-128 | rev)
```

Comment:
I have been using `${CODEBUILD_BUILD_ID##*:}` to simply remove the leading part of the build id.

Reply:
I adopted this. thanks.